package.json

```diff
@@ -33,6 +33,7 @@
     "babel-plugin-transform-define": "2.0.0",
     "bootstrap": "4.6.0",
     "cheerio": "1.0.0-rc.9",
+    "extract-domain": "^2.3.3",
```

Comment:
Might not be needed if you move route of getting the domain from the email address